api/lib/index.js

```diff
@@ -18,6 +18,20 @@ module.exports = {
       },
     });
   },
+  isListOfNumbers: str => {
+    if (typeof str !== 'string' || Number.isNaN(str)) {
```

Comment:
the second condition will always be false if the first one is false. do you need the isNaN check anymore? the regex will catch non numbers.

Reply:
want it to catch things that are neither strings nor numbers.

Reply:
you don't need the `isNaN`.` typeof str !== 'string'` catches everything that is not a string. 
![image](https://user-images.githubusercontent.com/2982784/70214275-092b0d80-16f0-11ea-9653-3e2272faf598.png)